scripts/zfs-tests.sh

```diff
@@ -239,8 +239,8 @@ create_links() {
 constrain_path() {
 	. "$STF_SUITE/include/commands.cfg"
 
-	SYSTEM_DIRS="/bin /sbin /usr/bin /usr/sbin"
-	SYSTEM_DIRS+=" /usr/local/bin /usr/local/sbin"
+	SYSTEM_DIRS="/usr/local/bin /usr/local/sbin"
```

Comment:
Can you add a comment explaining that the specific search order here is important for FreeBSD and why.